src/project/view/exportdialogmodel.cpp

```diff
@@ -94,7 +94,8 @@ ExportDialogModel::ExportDialogModel(QObject* parent)
         ExportType::makeWithSubtypes(musicXmlTypes,
                                      qtrc("project/export", "MusicXML")),
         ExportType::makeWithSuffixes({ "brf" },
-                                     qtrc("project/export", "Braille"),
+                                     //: It's called "bar-over-bar" even in US English, not "measure-over-measure".
```

Comment:
But is the meaning still like "measure over measure"?
And I googled a bit and it seems there's also a difference between bar-over-bar and bar-by-bar. Might be good to mention that as well.